packages/playwright-core/ThirdPartyNotices.txt

```diff
@@ -1229,6 +2813,6 @@ END OF yazl@2.5.1 AND INFORMATION
 
 SUMMARY BEGIN HERE
 =========================================
-Total Packages: 48
+Total Packages: 60
```

Comment:
We are paying a 25% bump in # of deps for a feature that does not link to a user report linked. Usually not a very good sign.

Reply:
True. It also increases the zip bundle size from 112kb to 202kb. We had this attempt of writing our own tar parser, maybe we should give it another try.